commonui/src/main/java/com/hyperwallet/android/ui/common/insight/HyperwalletInsight.java

```diff
@@ -221,12 +231,20 @@ public Handler getHandler() {
     public void trackError(@NonNull final Context context, @NonNull final String pageName,
             @NonNull final String pageGroup, @NonNull final Map<String, String> errorInfoMap) {
 
+        HashMap<String, String> params = new HashMap<>(2);
+        for (Map.Entry<String, String> entry : errorInfoMap.entrySet()) {
+            if (!ERROR_VALUES.contains(entry.getKey())) {
+                params.put(entry.getKey(), entry.getValue());
+            }
+        }
```

Comment:
I'm not a fan of this, but not sure how else to do it. Personally, I think we should get rid of ErrorInfo object in favour of a Map so we can take advantage of loose typing. Since in end, they all just get thrown into "event_params"